docs/tutorials/sml-classification.ipynb

```diff
@@ -0,0 +1,818 @@
+    "\n",
+    "In this step, we explore several configurations of the quantum ensemble to understand how key parameters -specifically `num_learners` and `regularization` - affect both result quality and QPU usage.\n",
+    "- `num_learners` determines the ensemble width (and implicitly, the number of qubits), influencing the model’s capacity and computational cost.\n",
+    "- `regularization` controls sparsity and overfitting, shaping how many learners remain active after optimization.\n",
+    "\n",
+    "By varying these parameters, we can see how ensemble width and regularization interact: increasing width typically improves recall and F1 but costs more QPU time, while stronger or adaptive regularization can improve generalization at roughly the same hardware footprint. The next subsections walk through three representative configurations to illustrate these effects."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8da41d4d-f06d-4b67-a8ee-b07fc0289558",
+   "metadata": {},
+   "source": [
+    "### Baseline\n",
+    "\n",
+    "This configuration uses `num_learners = 30` and `regularization = 7`.\n",
+    "\n",
+    "- `num_learners` controls the ensemble width — effectively the number of weak learners combined and, on quantum hardware, the **number of qubits required**. A larger value expands the combinatorial search space and can improve accuracy and recall, but also increases circuit width, compilation time, and overall QPU usage.\n",
+    "- `regularization` sets the penalty strength for including additional learners. With the default \"onsite\" regularization, higher values enforce stronger sparsity (fewer learners kept), while lower values allow more complex ensembles.\n",
+    "\n",
+    "This setup provides a low-cost baseline, showing how a small ensemble behaves before scaling width or tuning sparsity."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "e702c986-fd7f-4ea1-a93c-8cd5ef0c4a0b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Problem scale and regularization\n",
+    "NUM_LEARNERS = 30\n",
+    "REGULARIZATION = 7"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "06c261c1-a3c1-42c9-a522-63f3fc01a970",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "-- Submitting quantum-enhanced ensemble job --\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
```

Comment:
I see a few warnings in some of the output cells. Will the user see the same warnings? Is it avoidable?